integration_test.go

```diff
@@ -135,7 +135,7 @@ func TestFsnotifyMultipleOperations(t *testing.T) {
 	}
 	f.Sync()
 
-	time.Sleep(time.Millisecond)
+	time.Sleep(50 * time.Millisecond)
```

Comment:
maybe it would be worth defining a `const`?

Reply:
Naming is hard. Any suggestions for what to name it? I'm happy to make that change.
I was searching incorrectly for `static` for a different line in the FEN code. What I wanted is `const`, so this was also serendipitous.

Reply:
Naming is hard. Brainstorming 🌩️ 

It's a `sleepDuration` ... but to be more specific, it's the time to wait for events.

It is a `timeout` of sorts, because we assume it failed after this time elapses.

Uhm... I don't know. What do you think?

Reply:
`shortNap` ?

Reply:
🤷🏼

Reply:
We also have a 500ms sleep which is for waiting for the events to finish rolling in that could also be refactored.
call that `longNap` while I'm in there?

Reply:
`eventSeparator` (50ms) and `waitForEvents` (500ms) or something like that could be a little more descriptive?

Reply:
I've implemented those last two, but they are easy string replacements with something better.